presto-google-sheets/src/main/java/com/facebook/presto/google/sheets/SheetsMetadata.java

```diff
@@ -100,25 +101,42 @@ public ConnectorTableLayout getTableLayout(ConnectorSession session, ConnectorTa
     @Override
     public ConnectorTableMetadata getTableMetadata(ConnectorSession session, ConnectorTableHandle table)
     {
-        Optional<ConnectorTableMetadata> connectorTableMetadata = getTableMetadata(((SheetsTableHandle) table).toSchemaTableName());
+        Optional<ConnectorTableMetadata> connectorTableMetadata = getTableMetadata(session, ((SheetsTableHandle) table).toSchemaTableName());
         if (!connectorTableMetadata.isPresent()) {
             throw new PrestoException(SHEETS_UNKNOWN_TABLE_ERROR, "Metadata not found for table " + ((SheetsTableHandle) table).getTableName());
         }
         return connectorTableMetadata.get();
     }
 
-    private Optional<ConnectorTableMetadata> getTableMetadata(SchemaTableName tableName)
+    private Optional<ConnectorTableMetadata> getTableMetadata(ConnectorSession session, SchemaTableName tableName)
     {
         if (!listSchemaNames().contains(tableName.getSchemaName())) {
             return Optional.empty();
         }
         Optional<SheetsTable> table = sheetsClient.getTable(tableName.getTableName());
         if (table.isPresent()) {
-            return Optional.of(new ConnectorTableMetadata(tableName, table.get().getColumnsMetadata()));
+            List<ColumnMetadata> columns = table.get().getColumnsMetadata().stream()
+                    .map(column -> normalizedColumnMetadata(session, column))
+                    .collect(toList());
+
+            return Optional.of(new ConnectorTableMetadata(tableName, columns));
         }
         return Optional.empty();
     }
 
+    private ColumnMetadata normalizedColumnMetadata(ConnectorSession session, ColumnMetadata columnMetadata)
+    {
+        return ColumnMetadata.builder()
+                .setName(normalizeIdentifier(session, columnMetadata.getName()))
+                .setType(columnMetadata.getType())
+                .setHidden(columnMetadata.isHidden())
+                .setNullable(columnMetadata.isNullable())
+                .setComment(columnMetadata.getComment().orElse(null))
+                .setProperties(columnMetadata.getProperties())
+                .setExtraInfo(columnMetadata.getExtraInfo().orElse(null))
+                .build();
```

Comment:
This seems like a reoccurring method. Wonder if we can add a utility method on ColumnMetadata which effectively does this and accepts a lambda or already-normalized name. Will reduce code duplication